ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -544,7 +544,7 @@ export const ImportTokensModal = ({ onClose }) => {
                                   history.push(
                                     `${SECURITY_ROUTE}#auto-detect-tokens`,
                                   );
-                                  history.push(mostRecentOverviewPage);
+                                  onClose();
```

Comment:
I would leave the modal onclose before redirecting, to avoid having modal on new settings page, wdyt?